dist/yaml/static/ovnkube-master.yaml

```diff
@@ -0,0 +1,474 @@
+# ovnkube-master
```

Comment:
The static daemonsets may turn into a problem keeping them in-sync with the templates. I originally resisted using templates because of the value of the checked in daemonset. When we went to templates, I deleted the static files. I think it is very worthwhile to keep the templates as the reference copy and use the dist/images/daemonset.sh to expand the daemonset  templates.

Reply:
I have explained the rationale for using static yamls in commit message (first commit). I think, we cannot have beginner triers clone repo and run make commands. The idea of daemonsets is to make it easy for people to try out the project.